docs/_docs/themes.md

```diff
@@ -35,13 +44,70 @@ Jekyll will look first to your site's content, before looking to the theme's def
 Refer to your selected theme's documentation and source repository for more information on what files you can override.
 {: .note .info}
 
-To locate theme's files on your computer, run `bundle show` followed by
-the name of the theme's gem, e.g. `bundle show minima` for default Jekyll's
-theme. Then copy the files you want to override, from the returned path to your root folder.
+To locate theme's files on your computer:
+
+1.  Run `bundle show` followed by the name of the theme's gem, e.g., `bundle show minima` for default Jekyll's theme. 
+    
+    The location of the theme gem is returned. For example, minima is located in `/usr/local/lib/ruby/gems/2.3.0/gems/minima-2.1.0` on a Mac.
+        
+2.  Change to the directory's location and open the directory in Finder or Explorer:
+
+    ```
+    cd /usr/local/lib/ruby/gems/2.3.0/gems/minima-2.1.0
+    open . 
+    # for Windows, use "explorer ."
+    ```
+    
+    A Finder or Explorer window opens showing the theme's files and directories.
+     
+    With a clear understanding of the theme's files, you can now override any theme file by creating a similarly named file in your Jekyll site directory.
+    
+If you want to get rid of the theme gem altogether, copy the files from the theme gem's directory into your Jekyll site directory (for example, copy them to `/myblog` if you created your Jekyll site at `/myblog`).
+
+Then modify the Gemfile and configuration to remove references to the theme gem. For example, to remove Minima:
+*  Open **Gemfile** and remove `gem "minima", "~> 2.0"`.
+*  Open **_config.yml** and remove `theme: minima`.
+    
+Now `bundle update` will no longer get updates for the theme gem.
+
+## Using themes other than the default
+
+The `jekyll new <PATH>` command isn't the only way to create a new Jekyll site with a gem-based theme. You can also [find gem-based themes](https://github.com/jekyll/jekyll/wiki/Themes) online and incorporate them into your Jekyll project. To install a gem-based theme:
+    
+1.  Add the theme to your site's `Gemfile`:
+
+    ```
+    gem 'my-awesome-jekyll-theme'
+    ```
+    
+3.  Install the theme:
+    
+    ```
+    bundle install
+    ```
+    
+3. Add the following to your site's `_config.yml` to activate the theme:
+    
+    ```
+    theme: my-awesome-jekyll-theme
+    ```
+
+5.  Build your site:
+    
+    ```
+    bundle exec jekyll serve
+    ```
+    
+You can have multiple themes listed in your site's Gemfile, but only one theme can be selected in your site's `_config.yml`.
+{: .note .info }
+
+If you're publishing your Jekyll site on [Github Pages](https://pages.github.com/), note that Github Pages supports only some gem-based themes. See [Supported Themes](https://pages.github.com/themes/) in Github's documentation to see which themes are supported. Currently only Minima is currently supported.
```

Comment:
updated. i'm glad to see the expanded support. when i made my edit, this was true. thankfully now there is more than just 1 gem theme.